src/js/components/AccordionPanel.js

```diff
@@ -42,8 +51,15 @@ export default class AccordionPanel extends Component {
       }
     );
 
+    let controlIcon;
+    if (icon) {
+      controlIcon = icon;
+    } else if (icon === undefined) {
+      controlIcon = <TabNextIcon className={`${CLASS_ROOT}__control`} />;
+    }
+
     return (
-      <ListItem className={classes} direction="column" pad="none">
+      <ListItem className={classes} direction="column" pad="none" {...props}>
```

Comment:
I think we should put `...props` first, otherwise, caller's passing 'direction' or 'pad' would override what we are setting.